grommet-leaflet/src/layers/Marker.jsx

```diff
@@ -3,14 +3,20 @@ import PropTypes from 'prop-types';
 import ReactDOMServer from 'react-dom/server';
 import { v4 as uuidv4 } from 'uuid';
 import { ThemeContext } from 'styled-components';
-import { Popup as LeafletPopup, Marker as LeafletMarker } from 'react-leaflet';
+import { Marker as LeafletMarker, Popup as LeafletPopup } from 'react-leaflet';
 import L from 'leaflet';
 import { Pin, Popup } from '.';
 
-const Marker = ({ children, icon, ...rest }) => {
+const Marker = ({ children, icon, popup: popupProp, ...rest }) => {
```

Comment:
Can we add popup to the propTypes at the bottom of this file? Also should we remove the `children` prop?

Reply:
Thank you, good catch. Adding.

Reply:
Done.